frappe/automation/doctype/auto_repeat/auto_repeat.py

```diff
@@ -325,7 +326,7 @@ def set_auto_repeat_as_completed():
 			doc.save()
 
 @frappe.whitelist()
-def make_auto_repeat(doctype, docname, frequency, start_date, end_date = None):
+def make_auto_repeat(doctype, docname, frequency = 'Daily', start_date = today(), end_date = None):
```

Comment:
Not a good idea to set dynamic values in default parameter values. Python will only set it once, it won't be called everytime the method is called.

```suggestion
def make_auto_repeat(doctype, docname, frequency = 'Daily', start_date = None, end_date = None):
```